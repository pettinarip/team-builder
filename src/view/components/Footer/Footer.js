import React from 'react'

const Footer = () => (
  <footer>
    <div className='footer-link'>
      <small>by</small> <a href='https://github.com/pettinarip' target='_blank'>@pettinarip</a>
    </div>
  </footer>
)

Footer.propTypes = {}

export default Footer
