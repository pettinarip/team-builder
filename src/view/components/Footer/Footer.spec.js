import React from 'react'
import { shallow } from 'enzyme'
import Footer from './Footer'

describe('Footer component', () => {
  it('should render the component properly', () => {
    const component = shallow(<Footer />)
    expect(component).toMatchSnapshot()
  })
})
