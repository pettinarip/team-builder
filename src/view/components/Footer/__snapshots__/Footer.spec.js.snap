// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render the component properly 1`] = `
<footer>
  <div
    className="footer-link"
  >
    <small>
      by
    </small>
     
    <a
      href="https://github.com/pettinarip"
      target="_blank"
    >
      @pettinarip
    </a>
  </div>
</footer>
`;
